scale/source/serializers.py

```diff
@@ -10,6 +10,9 @@ class SourceFileBaseSerializer(ScaleFileBaseSerializer):
     is_parsed = serializers.BooleanField()
     parsed = serializers.DateTimeField()
 
+    is_superseded = serializers.BooleanField()
+    superseded = serializers.DateTimeField()
+
```

Comment:
Remove these. We don't currently support superseding as a concept for source files.